src/main/java/seedu/address/logic/parser/EditCommandParser.java

```diff
@@ -0,0 +1,75 @@
+package seedu.address.logic.parser;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.EditAttributeCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+
+/**
+ * Parses user input into an EditAttributeCommand for managing person attributes.
+ * This parser converts user input strings into an EditAttributeCommand object
+ * that can edit attributes associated with a person in the address book.
+ */
+public class EditCommandParser implements Parser<EditAttributeCommand> {
+
+    /**
+     * Parses the given {@code String} of arguments in the context of the EditAttributeCommand
+     * and returns an EditAttributeCommand object for execution.
+     * Command format: editAttribute /uuid /attributeName1 attributeValue1 /attributeName2 attributeValue2 ...
+     * Examples:
+     * editAttribute /4000 /Name John Doe /Phone 12345678
+     */
+    public EditAttributeCommand parse(String args) throws ParseException {
+        args = args.trim();
+
+        // Split the arguments string on spaces after the first "/"
+        String[] parts = args.split(" ", 3);
+
+        // Validate command structure
+        if (parts.length < 3 || !"editAttribute".equalsIgnoreCase(parts[0])) {
+            throw new ParseException("Invalid command format. Expected format: editAttribute /uuid [attributes]");
```

Comment:
Could this use MESSAGE_USAGE, or some other constant?

Reply:
yep